test/bin/run_api_tests.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -eu
+
+declare apitest_env_file
+declare bearer_token
+
+declare OKTA_URL
+declare OKTA_USERNAME
+declare OKTA_PASSWORD
+
+_get_osmt_project_dir() {
+  local project_dir; project_dir="$(git rev-parse --show-toplevel 2> /dev/null)"
+  echo "$project_dir"
+}
+
+function source_osmt_apitest_env_file() {
+  echo "Sourcing $apitest_env_file"
+  set -o allexport
+  # shellcheck source="test/osmt-apitest.env"
+  source "$apitest_env_file"
+  set +o allexport
+}
+
+function get_bearer_token() {
+  declare auth_env; auth_env="$test_dir/postman/osmt-auth.environment.json"
+
+	# Running postman collections
+	echo "Getting access token from OKTA ..."
+  npx "$test_dir/node_modules/.bin/newman" \
+    run "$test_dir/postman/osmt-auth.postman_collection.json" \
+      --env-var oktaUsername="$OKTA_USERNAME" \
+      --env-var oktaPassword="$OKTA_PASSWORD" \
+      --env-var oktaUrl="$OKTA_URL" \
+      --env-var baseUrl="$BASE_URL" \
+      --ignore-redirects \
+      --export-environment "$auth_env"
+
+  bearer_token="$(jq '.["values"][] | select(.key=="bearerToken") | .value' "$auth_env")"
```

Comment:
I didn't have `jq` command installed, do we need to consider install the command in this script before is used?